### DISCUSSION TITLE
The file uploaded to s3 using UploadFile is corrupted

### DISCUSSION THREAD

**Discussion by ali-gulzar** [fastapi/fastapi#9400] [Questions]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [ ] I commit to help with one of those options 👆

### Example Code

```python
from fastapi import APIRouter, UploadFile, status
import boto3

router = APIRouter()

def upload_image(image: UploadFile):
    s3_client =  boto3.client("s3")
    try:
        s3_client.put_object(Bucket='xxxx', Key=image.filename, Body=image.file.read())
    except Exception as e:
        raise HTTPException(detail=e, status_code=status.HTTP_400_BAD_REQUEST)

    s3_image_url = f'https://xxxx.s3.amazonaws.com/{image.filename}'
    return s3_image_url


@router.post("/infer-ingredient")
def infer_ingredient(image: UploadFile):
    image_url = s3_service.upload_image(image)
    return image_url
```

### Description

I uploaded an image using UploadFile in fastapi and the upload to s3 is successfull but when downloading the file, it says it is corrupted. When also trying to save the file with the contents=image.file.read() locally, the file is also corrupted. Is there some issue with the bytes? 

### Operating System

macOS

### FastAPI Version

0.93.0

### Python Version

3.9.15

**Comment by dragin-martin** [comment]
I AM HAVING SAME PROBLEM!!!! DID YOU FIGURE IT OUT?

**Comment by fmelihh** [comment]
Hi @ali-gulzar @dragin-martin I didn't test it in my locale, but could you try this ? 

```
from fastapi import APIRouter, UploadFile, status, HTTPException
import boto3

router = APIRouter()


def upload_image(filename: str, image_bytes: bytes):
    s3_client = boto3.client("s3")
    try:
        s3_client.put_object(Bucket="xxxx", Key=filename, Body=image_bytes)
    except Exception as e:
        raise HTTPException(detail=e, status_code=status.HTTP_400_BAD_REQUEST)

    s3_image_url = f"https://xxxx.s3.amazonaws.com/{filename}"
    return s3_image_url


@router.post("/infer-ingredient")
async def infer_ingredient(image: UploadFile):
    image_bytes = await image.read()
    image_url = upload_image(filename=image.filename, image_bytes=image_bytes)
    return image_url

```

**Comment by dragin-martin** [comment]
I figured it out.
use file.file.seek(0) to rewind the pointer once it is read!!!!!

otherwise you will have serious issues!!
